### PR TITLE
[AC-4629] - set Proxy API calls to Test-3's Host

### PR DESCRIPTION
**Pull request by yotammanor**
The current settings redirect Proxy API calls to production, instead of test.
This change solves this.

### CODE REVIEW

File: web/impact/impact/settings.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ class Base(Configuration):
     CORS_ORIGIN_ALLOW_ALL = True
     # settings.py
     REST_PROXY = {
-        'HOST': 'https://accelerate.masschallenge.org',
+        'HOST': 'https://accelerate-test-3-sf-elb-1843213725.us-east-1.elb.amazonaws.com',
```
Comment (shanbady) [inline web/impact/impact/settings.py]: only suggestion here is to have this derived from the environment as a separate ACCELERATE_SITE_URL setting similar to how we configure the database ALLOWED hosts.

**Review by shanbady — CHANGES_REQUESTED** [review]

File: web/impact/impact/settings.py (hunk 1 above)
Comment (yotammanor) [reply web/impact/impact/settings.py]: Good idea.

Comment (yotammanor) [reply web/impact/impact/settings.py]: Added ACCELERATE_SITE_URL to travis's settings with TEST3's url.

File: web/impact/impact/settings.py [hunk 2]
```diff
@@ -157,7 +156,7 @@ class Base(Configuration):
     CORS_ORIGIN_ALLOW_ALL = True
     # settings.py
     REST_PROXY = {
-        'HOST': 'https://accelerate.masschallenge.org',
+        'HOST': bytes(os.environ.get('ACCELERATE_SITE_URL'), 'utf-8'),
```
Comment (shanbady) [inline web/impact/impact/settings.py]: we should probably default this to the production accelerate site

**Review by shanbady — CHANGES_REQUESTED** [review]

**Review by shanbady — APPROVED** [review]

**Pull request merged**